iota-agent/src/main/java/org/iotacontrolcenter/iota/agent/action/util/AgentUtil.java

```diff
@@ -9,41 +9,22 @@
 
 public class AgentUtil {
 
-    public static boolean dirExists(String dirPath) {
-        boolean rval = false;
-        try {
-            File f = new File(dirPath);
-            rval = f.exists() && f.isDirectory();
-        }
-        catch(Exception e) {
-            System.out.println("dirExists(" + dirPath + ") exception: " + e.getLocalizedMessage());
-        }
-        return rval;
+    public static boolean dirDoesNotExist(String dirPath) {
+        File f = new File(dirPath);
+        return !f.exists() || !f.isDirectory();
```

Comment:
Might just be me, but this boolean expression is a little hard to read. I'd rewrite it as:

`!(f.exists() && f.isDirectory())`

The two give the same result, but this seems easier to read.